unionml/model.py

```diff
@@ -533,11 +533,14 @@ def _remote(self) -> Optional[FlyteRemote]:
         )
         return self.__remote__
 
-    def remote_deploy(self):
+    def remote_deploy(self, ignore_diff: bool = False):
         """Deploy model services to a Flyte backend."""
+        if self._remote is None:
+            raise RuntimeError("First configure the remote client with the `Model.remote` method")
+
```

Comment:
`mypy` was warning about this in the few lines below, a remote might not be configured. Found this pattern and message in some other functions.